File: PRAK503/PRAK503_2310817110011_Ghani-Mudzakir.py
def maksimal (a, b):
    c =a
    if a>b:
        c =a
    elif b>a:
        c = b
    
    return c

def minimal (a,b):
    d = a
    if a>b:
        d = b
    elif b>a:
        d = a
    
    return d

maks = -1000
min = 1000
banyaknya_input = input()

banyaknya_input = int(banyaknya_input)
for _ in range(banyaknya_input):


    nilai = input()
    nilai = int(nilai)
    maks = maksimal(maks, nilai)
    min = minimal(min, nilai)

print(f"{maks} {min}")

File: PRAK503/PRAK503_2310817110011_Ghani-Mudzakir.c
#include <stdio.h>

int maksimal(int a, int b){
    int c = a;
    if (b>a)
    {
        c = b;
    }
    else if (a>b)
    {
        c = a;
    }
    return c;
}
int minimal (int a, int b){
    int d = a;
    if (b>a)
    {
        d = a;
    }
    else if (a>b)
    {
        d = b;
    }
    return d;

}

int main()
{
    int batas = 0;
    int maks = -100000;
    int minim = 100000;
    int bilangan;
    scanf("%d", &bilangan);
    while(batas < bilangan)
    {
        int nilai;
        scanf("%d", &nilai); 
        maks = maksimal(maks, nilai);
        minim = minimal(minim, nilai);
        batas++;
    }
    printf("\n%d %d",maks,minim);
 
    return 0;
}

